helpers/test.py

```diff
@@ -51,7 +51,7 @@ def run_tests(self):
             import pytest
             if not self.pytest_args:
                 self.pytest_args = ['sherpa',]
-            errno = pytest.main(self.pytest_args)
+            errno = pytest.main([self.pytest_args,])
```

Comment:
I thought about that right after I committed. It's probably not right, let's see what Travis thinks. I'll remove the commit if that's the case.